core/src/main/java/tech/pegasys/eth2signer/core/multikey/metadata/ArtifactSignerFactory.java

```diff
@@ -74,7 +74,28 @@ public ArtifactSigner create(final FileKeyStoreMetadata fileKeyStoreMetadata) {
     }
   }
 
-  public ArtifactSigner create(final HashicorpSigningMetadata metadata) {
+  public ArtifactSigner create(final HashicorpSigningMetadata hashicorpMetadata) {
+    try (TimingContext ignored = privateKeyRetrievalTimer.labels("hashicorp").startTimer()) {
+      return createHashicorpArtifact(hashicorpMetadata);
+    }
+  }
+
+  private ArtifactSigner createKeystoreArtifact(final FileKeyStoreMetadata fileKeyStoreMetadata) {
```

Comment:
nit: should this method be remain below createHashicorpArtifact?

Reply:
I think I prefer all the private functions together after the public functions. So that's what I've done here.

Reply:
👍